htdocs/web_portal/controllers/downtime/add_downtime.php

```diff
@@ -158,28 +158,50 @@ function draw(\User $user = null)
         die(json_encode(array('UTC', 0)));
     }
 
-    // URL Mapping for `site` and `se`
+    /**
+     * URL Mapping for `site` and `se`.
+     *
+     * If the user wants to add a downtime to a specific site and SE,
+     * show only that site SEs. 
+     */
     elseif (isset($_GET['site']) && isset($_GET['se'])) {
         displaySiteAndSeEndpoints($user, $nowUtcDateTime);
     }
 
-    // URL Mapping for `site` ONLY
+    /**
+     * URL Mapping for `site` ONLY.
+     *
+     * If the user wants to add a downtime to a specific site,
+     * show all of the site SEs.
+     */
     elseif (isset($_GET['site'])) {
         displaySiteEndpoints($user, $nowUtcDateTime);
     }
 
-    // URL Mapping for `se` ONLY
+    /**
+     * URL Mapping for `se` ONLY.
+     *
+     * If the user wants to add a downtime to a specific SE,
+     * show only that service endpoints.
+     * 
+     * TODO - Remove the dead code.
+     */
     elseif (isset($_GET['se'])) {
         displaySeEndpoints($user, $nowUtcDateTime);
     }
 
-    // Generic URL mapping
+    /**
+     * Generic URL mapping.
+     * 
+     * User should be able to see all the service endpoints associated with the site selected.
+     * TODO - Default selection of the value in the `Select Affected Site` section should be consistent.
```

Comment:
Which part of the code does this refer to?

Reply:
![SiteNames](https://github.com/GOCDB/gocdb/assets/127398882/205ac8b8-46ff-4638-aaa7-60ec29a9cb09)

Default value in the Select Affected site Section: "This requires a team design discussion"

Reply:
Something to discuss at the next dev meeting :)